the_destination_of_shadow/DirectX.cpp

```diff
@@ -78,6 +113,54 @@ HRESULT DirectX::InitD3Device(HWND hWnd, CONST TCHAR* FilePath)
 	return S_OK;
 }
 
+KEY_STATE DirectX::GetInputState(BYTE curr_diks, BYTE prev_diks)
+{
+	if (IsKeyPressed(curr_diks)==TRUE)
+	{
+		return (prev_diks == OFF) ? PRESS : ON;
+	}
+	else
+	{
+		return (prev_diks == ON) ? RELEASE : OFF;
+	}
+}
+
+BOOL DirectX::IsKeyPressed(BYTE curr_diks)
+{
+	return (curr_diks & mask.NUM) ? TRUE : FALSE;
```

Comment:
こういうパッと見ナゾなコードには、
```suggestion
	// キーが押されているときはcurr_diksの最上位ビットが立っている
	return (curr_diks & mask.NUM) ? TRUE : FALSE;
```
等、コメントしておくと良いと思いますよ。

Reply:
了解です